cdap-docs/developers-manual/source/building-blocks/schedules.rst

```diff
@@ -18,6 +18,14 @@ class contains static methods to create schedules based on time, or schedules ba
 The name of a schedule must be unique in the application that it is in; the same name can
 be used in different applications.
 
+Schedules can be controlled by the :ref:`CDAP CLI <cli>` and the :ref:`Lifecycle HTTP
+RESTful API <http-restful-api-lifecycle>`. The :ref:`status of a schedule
+<http-restful-api-lifecycle-start-stop-status>` can be retrieved, and individual schedules
+:ref:`resumed or suspended <http-restful-api-lifecycle-schedules-suspend-resume>`. 
+
+When a schedule is initially deployed, it is in a *suspended* state; a command needs to be
```

Comment:
No, just because the UI doesn't typeset text correctly doesn't mean we have to here. :)